### ISSUE TITLE
Menu shows media/tools/settings items for user wihout permission

### ISSUE THREAD

**Issue by pjeutr** [thedevdojo/voyager#1168]
- Laravel Version: 5.4.19
- Voyager Version: 0.11.9
- PHP Version: 5.6.2
- Database Driver & Version:

### Description:
After I updated from 0.10 to 0.11, a custom role I created shows the media/tools/settings in the admin menu. In the created role I only ticked _Browse Admin_ to make the dashboard viewable. 
A click gives me an UnauthorizedHttpException so the system knows the user isn't authorized
Are more people seeing this?

![screen shot 2017-04-23 at 23 33 04](https://cloud.githubusercontent.com/assets/611968/25317661/4be2238e-287d-11e7-9ad3-d43c2b6e4440.png)
![screen shot 2017-04-23 at 23 34 15](https://cloud.githubusercontent.com/assets/611968/25317666/659d2bfc-287d-11e7-94b7-32225e833fbb.png)

### Steps To Reproduce:
Add a role, only tick _Browse Admin. 
Add the role to an user and login.

**Comment by nunomsh** [comment]
It shows, but wont open. I had the same problem, and the solution was to create different menus for each role, and load the menu according the user permissions. 
Something like

```
switch (Auth::user()->role_id) 
{
            case 1:
                echo menu('admin','admin_menu');
                break;
            case 3:
                echo menu('manager','admin_menu');
                break;
            case 5:
                echo menu('seller','admin_menu');
                break;
            default:
            echo menu('seller','admin_menu');
}
```

**Comment by pjeutr** [comment]
I dove a little deeper and removed 
`&& is_null($value->table_name) `
in admin_menu.blade.php:64
Now things work for me again, but it's in Voyager code...

Can't really figure out how it should work, But there seems to be something wrong with permissions, maybe related to https://github.com/the-control-group/voyager/issues/1172

**Comment by marktopper** [comment]
This is fixed in #1285 for `v1.0-pre-release` since this is a breaking change.
